src/pages/home/report/ReportActionItem.js

```diff
@@ -561,8 +562,9 @@ function ReportActionItem(props) {
                         {props.shouldDisplayNewMarker && <UnreadActionIndicator reportActionID={props.action.reportActionID} />}
                         <MiniReportActionContextMenu
                             reportID={props.report.reportID}
-                            reportAction={props.action}
-                            isArchivedRoom={ReportUtils.isArchivedRoom(originalReport)}
+                            reportActionID={props.action.reportActionID}
+                            originalReportID={originalReportID}
+                            isArchivedRoom={ReportUtils.isArchivedRoom(props.report)}
```

Comment:
@dukenv0307 

We are using `report` to check `isArchivedRoom` here in the `MiniReportActionContextMenu` but `originalReport/originalReportID` to check `isArchivedRoom` in `ReportActionContextMenu.showContextMenu`. Was this a mistake or was there some logic in this?